.github/workflows/tests.yml

```diff
@@ -11,20 +11,20 @@ jobs:
     name: Check the code style
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - uses: actions/setup-python@v4
       with:
         python-version: "3.10"
     - uses: pre-commit/action@v3.0.0
 
   tests:
-    name: Run the tests
+    name: Run python tests
```

Comment:
Oh, also, this change looks like it requires a branch protection rules change.  In other words, we need to make sure that the newly named Python and Rust test runs are performed and passing before merging in to `main`.

Reply:
Ah, good catch! Thank you for taking a look here